### ISSUE TITLE
error from ListPipeline: line 1.966: unknown field name "input" in pps.Input

### ISSUE THREAD

**Issue by emk** [pachyderm/pachyderm#1801]
Upgraded to 1.4.6, starting from the `master` build you sent us a while ago:

```
> pachctl version
COMPONENT           VERSION             
pachctl             1.4.6               
pachd               1.4.6               
> pachctl list-pipeline
error from ListPipeline: line 1.966: unknown field name "input" in pps.Input
> pachctl list-job
error from InspectJob: line 1.1034: unknown field name "input" in pps.Input
```

**Comment by emk** [comment]
This also breaks `delete-all`.

The only workaround I've found is to delete _everything_ manually, including the S3 bucket and underlying `etcd` volume.

**Comment by derekchiang** [comment]
@emk I think you likely just need to upgrade your `pachctl` as well.  Use `pachctl version` to see which version of `pachctl` you are using.

**Comment by derekchiang** [comment]
My bad I didn't see that you did `pachctl version` already.  I don't know what's going on then.

**Comment by jdoliner** [comment]
I believe this is a fairly isolated issue caused by the fact that there was some data lying around which was created by the 1.4.6 RC which had a slightly different schema from the actual 1.4.6 release. 1.4.6 should migrate smoothly from 1.4.5 but not from the RC state unfortunately.

**Comment by emk** [comment]
Yeah, I spent about half a day running a recursive S3 delete to clear out all our Pachyderm FS and rebuild from scratch.

**Comment by jdoliner** [comment]
I believe this issue was due to the fact that we cut an RC for @emk before we released 1.4.6 publicly and the schema got tweaked in between. Sorry for the annoyance on this one @emk. Since this was an isolated issue that shouldn't be a problem again since I don't think anyone will be using that RC again I think I'm going to close this issue.
